### PR TITLE
[MAINT] remove code climate and update badges

### PR DESCRIPTION
**Pull request by Remi-Gau**
- remove code climate config
- update badges in README 
- update links to legacy validator in README

### CODE REVIEW

File: bids-validator/README.md [hunk 1]
```diff
@@ -335,6 +350,9 @@ see available options.
 
 ## Docker image
 
+<!-- TODO update version -->
+**:rotating_light: docker images of the bids-validator used the legacy validator until version 1.X.Y**
```
Comment (Remi-Gau) [inline bids-validator/README.md]: not sure which version of the docker image contains the new validator.

Comment (effigies) [reply bids-validator/README.md]: I don't think any do.

File: bids-validator/README.md [hunk 2]
```diff
@@ -335,6 +350,9 @@ see available options.
 
 ## Docker image
 
+<!-- TODO update version -->
+**:rotating_light: at this time (november 2024) docker images of the bids-validator still use the legacy validator. But this should soon change.**
+
```
Comment (Remi-Gau) [inline bids-validator/README.md]: changed the wording to let users know that they may still be using the legacy validator if using the docker image

File: bids-validator/README.md [hunk 3]
```diff
@@ -335,6 +350,9 @@ see available options.
 
 ## Docker image
 
+<!-- TODO update version -->
+**:rotating_light: at this time (november 2024) docker images of the bids-validator still use the legacy validator. But this should soon change.**
```
Comment (Remi-Gau) [inline bids-validator/README.md]:
```suggestion
**:rotating_light: at this time (November 2024) docker images of the bids-validator still use the legacy validator. But this should soon change.**
```

**Pull request merged**